assets/components/ReportCardContainer.js

```diff
@@ -33,17 +34,196 @@ class ReportCardContainer {
     return DateFormat.getTime(new Date(this.created_at));
   }
 
+  getTopControls() {
+    const contentHTML = `
+
+    <div class="circle-border pointer" id="reportCloseBtn">
+      <i
+        class="icon-close-square"
+        style="background-color: var(--neutral-500)"
+      ></i>
+    </div>
+    <div class="circle-border pointer">
+      <i class="icon-share" style="background-color: var(--neutral-500)"></i>
+    </div>
+`;
+    const div = document.createElement('div');
+    div.setAttribute('class', 'report-card__top-controls');
+    div.innerHTML = contentHTML;
+    return div;
+  }
+
+  getHeading() {
+    const icon = this.settings?.detail;
+    const contentHTML = `
+			<span class="btn__icon report-card__heading__icon" style="background-color: ${icon.iconBackround}">
+				<i class="${icon.icon}" style="width:1.5rem; height:1.5rem; background-color: white"></i>
```

Comment:
var(--white)

Reply:
addressed in: https://github.com/whamester/WMDD-4885-Project/pull/122/commits/6c47bbcbff6ca76a64e9ecb9aa9e812b90ca0a40